packages/api/src/external/carequality/command/cq-directory/get-organizations-for-xcpd.ts

```diff
@@ -1,30 +0,0 @@
-import { CQDirectoryEntryModel } from "../../models/cq-directory";
-import {
-  getRecordLocatorServiceOrganizations,
-  getStandaloneOrganizations,
-  getSublinkOrganizations,
-} from "./cq-gateways";
-
-function sortOrgsByNearbyOrder(orgs: CQDirectoryEntryModel[], orderMap: Map<string, number>) {
```

Comment:
Moved to https://github.com/metriport/metriport/pull/2974/files#diff-4d528d0967a22a55b28f15970217a7615b95e02894fa6cd0302611827666a396R122